libs/discordURL.ts

```diff
@@ -0,0 +1,30 @@
+export const DISCORD_OAUTH_URL = 'https://discord.com/api/oauth2/token';
+
+export const discordRedirectBaseUrl = `${process.env.NEXT_PUBLIC_DISCORD_REDIRECT_URI_BASE}/api/discord/auth`
+
+export const discordScope = ["identify", "guilds"].join(" ");
+
+export const DISCORD_CLIENT_ID = process.env.NEXT_PUBLIC_DISCORD_CLIENT_ID
```

Comment:
we should update related files if we introduce new process env:

- [ ] Update README to introduce
- [ ] Add new entry to example.env

Also to keep in mind that we need to write this env into Vercel before the deployment.